### DISCUSSION TITLE
Project Polishing

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#227] [General]
I have been doing a lot of user testing on my memory game this week. I had family and friends test it and I discovered some bugs that need to be worked out. I think that I am 85-90 percent finished, but the user testing this week helped to point out a few issues that my game has. I have been doing a lot of outside research with watching youtube videos and reviewing exercises from class to address the problems that I am running in to. I am confident that it will be completed well before the final turn in, but I still have a little bit that I need to do to finalize my game design.

**Comment by beto0890** [comment]
Great job on your memory game! it's looking great. Hopefully, you will find the answers to your bug problems. I will let you know if I find something.

**Comment by CJ3315** [comment]
User testing is a big component! That's awesome that you did that! Other than the bugs did they find the game helpful/entertaining?
